src/Init/Meta.lean

```diff
@@ -1255,6 +1255,8 @@ structure Config where
     `simp` to visit them. If `dsimp := false`, then argument is not visited.
   -/
   dsimp             : Bool := true
+  /-- This check is implemented in `Lean.Meta.Tactic.Simp.Main`, in `simpGoal`. -/
```

Comment:
the comment here should describe what the flag does, not where it is implemented

Reply:
Changed. My concern was that with so many entry points into the `simp` machinery, it might be part of the specification to say which entry points were affected...